ontology/cdao.md

```diff
@@ -21,6 +21,7 @@ license:
 activity_status: active
 repository: https://github.com/evoinfo/cdao
 preferredPrefix: CDAO
+domain: organisms
```

Comment:
@balhoff thoughts?

Reply:
Same comment here as above, BTW. It is quite unfortunate that the science of investigating why biology is what biology is isn't available as a choice here. Seems like a bad oversight to me.

Reply:
@hlapp feel free to request a new category on the issue tracker! Definitely open to add!